custom/enikshay/nikshay_datamigration/migrations/0010_followup_changes.py

```diff
@@ -2,6 +2,7 @@
 # Generated by Django 1.10.7 on 2017-04-27 18:33
 from __future__ import unicode_literals
 
+from __future__ import absolute_import
```

Comment:
nit, but could these be combined with the previous `from __future__` import?

Reply:
yeah, but it's much easier to run the script to add them and leave as is.  if every file in the codebase ends up having both I'll get carpal tunnel from combining them all.

Reply:
in the long-run they'll get removed anyway

Reply:
lol, ok.